executor/status_handler.go

```diff
@@ -107,19 +108,34 @@ func (executor *Executor) podsStatusHandler(entityName string, namespace string,
 func (executor *Executor) deploymentsHandler(entityName string, namespace string) (deploymentName string, targetPods int32, err error) {
 	replicasets, err := executor.kube.GetNamespaceReplicaSets(namespace)
 
-	if err != nil {
```

Comment:
Why was this removed

Reply:
no need to double check for the replicaset name, we fetch the right one before the short pooling loop, so every time we will get the same replicaset so no need for this additional request